classes/core/PKPContainer.inc.php

```diff
@@ -263,7 +272,7 @@ protected function loadConfiguration()
      */
     public function basePath($path = '')
     {
-        return $this->basePath . ($path ? "/$path" : $path);
+        return $this->basePath . ($path ? "/${path}" : $path);
```

Comment:
This form of string interpolation will be deprecated: https://wiki.php.net/rfc/deprecate_dollar_brace_string_interpolation